bullet_train/lib/bullet_train/resolver.rb

```diff
@@ -251,10 +251,71 @@ def locale_path
       nil
     end
 
+    # In this search, we prioritize files in local themes,
+    # and then look in theme gems if nothing is found.
+    def js_or_stylesheet_path
+      file_name = @needle.split("/").last
+
+      # Prioritize the current theme, and fall back to
+      # the default `light` theme if nothing is found locally.
+      puts "Searching under current theme: #{current_theme.blue}"
+
+      # If the file name doesn't include the theme name (i.e. - light.tailwind.css),
+      # we append the theme name to the path to match something like "app/assets/javascript/foo/".
+      append_theme_name = !file_name.match?("#{current_theme}.")
+      asset_globs = ["js", "css"].map do |extention|
+        path = "app/assets/"
+        if append_theme_name
+          path += case extention
+          when "js"
+            "javascript/"
+          when "css"
+            "stylesheets/"
+          end
+          path += "#{current_theme}/"
+        end
+        path + "**/*.#{extention}"
+      end
+
+      # Ensure we include JavaScript files that exist under the app's root directory.
+      asset_globs << "*.js"
```

Comment:
I find this code pretty complex to read, could we replace it with something like this? (If I haven't gotten it right, I'd say that's another indicator that it's too complex and we should simplify). 

```suggestion
      asset_globs = [
        "*.js", # Include JavaScript files under the app's root directory.
        "app/assets/javascript/**/*.#{current_theme}.js",
        "app/assets/javascript/#{current_theme}/**/*.js",
        "app/assets/stylesheets/**/*.#{current_theme}.css",
        "app/assets/stylesheets/#{current_theme}/**/*.css",
      ]
```

Reply:
Yup, this is much simpler, too much overthinking on my part.